src/System.Net.WebSockets.Client/src/System/Net/WebSockets/WinHttpWebSocketCallback.cs

```diff
@@ -135,7 +135,7 @@ private static void OnRequestHandleClosing(WinHttpWebSocketState state)
 
             state.RequestHandle.DetachCallback();
             state.RequestHandle = null;
-
+            
```

Comment:
looks like you added more whitespace?

Reply:
Not really. The first commit removed extra trailing whitespace what was there. In the second commit, I reverted that file to its original state as is in `master`, because there were no changes. So this extra trailing whitespace is back again. In this PR, there is only one file changed now - the resources.

Reply:
ok. thx.